checkov/terraform/checks/provider/awscc/credentials.py

```diff
@@ -0,0 +1,37 @@
+import re
+from typing import Any, Dict, List
+
+from checkov.common.models.consts import access_key_pattern, secret_key_pattern
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.provider.base_check import BaseProviderCheck
+
+
+class AWSCCCredentials(BaseProviderCheck):
+    def __init__(self) -> None:
+        name = "Ensure no hard coded AWS access key and secret key exists in provider"
+        id = "CKV_AWSCC_41"
```

Comment:
```suggestion
        id = "CKV_AWS_41"
```
The middle value is the service provider